cmd/apprepository-controller/controller.go

```diff
@@ -101,7 +101,7 @@ func NewController(
 	apprepoclientset clientset.Interface,
 	kubeInformerFactory kubeinformers.SharedInformerFactory,
 	apprepoInformerFactory informers.SharedInformerFactory,
-	kubeappsNamespace string) *Controller {
+	conf *Config) *Controller {
```

Comment:
> Is there a reason to pass by ref here, rather than by value? (other than optimization which I don't think is significant here).

I am fine using refs (pointers) as default when using structs